### DISCUSSION TITLE
Opening Positron on MacOS promts me to enter keychain password.

### DISCUSSION THREAD

**Discussion by alan-sims** [posit-dev/positron#5769] [Q&A] [answered]
Whenever I open positron I get the following dialogue box:
_Positron wants to use your confidential information stored in "Positron Safe Storage" in your keychain. To allow this, enter the "login" keychain password._
Just wanted to check that this might be from Positron trying to fetch my github password which is stored in mac keychain, as I have positron and github linked.

**Comment by juliasilge** [comment, marked as answer]
Thanks for this report @alan-sims! We are tracking how this dialog gets surfaced to users in https://github.com/posit-dev/positron/issues/2274, and you can specifically see some details about how it's being called in https://github.com/posit-dev/positron/issues/2274#issuecomment-1957789885. Positron (and VS Code upstream) use Electron's "safe storage" to store things that shouldn't be kept in plain text. We do know it's a fairly confusing modal and are working to see how better to handle it.
